doc/introduction/configuration.rst

```diff
@@ -0,0 +1,128 @@
+Configuration
+=============
+
+Configuring your account
+------------------------
+
+Before using the Xanadu cloud platform, you will need to configure your credentials. This
+can be done using several methods:
+
+* **Configuration file**. Use the Strawberry Fields :ref:`command
+  line interface <cli>`,
+
+  .. code-block:: bash
+
+      $ sf configure --token MYAUTH
+
+  or the :func:`~.store_account` function,
+
+  >>> sf.store_account("MyToken")
+
+  to create a configuration file containing your credentials. This is the recommended approach.
+
+  By default, the account is stored in a ``config.toml`` file located in the
+  *Strawberry Fields configuration directory*. For more detailed examples, visit the
+  :func:`~.store_account` documentation.
+
+* **Environment variables**. Set the ``SF_API_AUTHENTICATION_TOKEN`` environment variable
+  with your API token.
+
+* **Keyword argument**. Pass the keyword argument ``token`` when initializing the
+  :class:`~.RemoteEngine`.
+
+
+.. _cli:
+
+Command line interface
+----------------------
+
+The Strawberry Fields command line program ``sf`` provides several utilities
+including:
+
+* ``sf configure [--token] [--local]``: configure the connection to the cloud platform
+
+* ``sf run input [--output FILE]``: submit and execute quantum programs from the command line
```

Comment:
Or somehow just to indicate that `input` is not a command
```suggestion
* ``sf run INFILE [--output OUTFILE]``: submit and execute quantum programs from the command line
```

Reply:
nice!